src/store/rosTypes.ts

```diff
@@ -0,0 +1,8 @@
+type messageType =
```

Comment:
This should be `MessageType` to follow TypeScript naming conventions.

Reply:
Good catch!